drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
```

Comment:
Not necessary. am just trying to understand the alternate proposal.
If you can provide a reasonable alternative implementation for the provided functionality, that is good enough.